server/index.js

```diff
@@ -86,14 +95,23 @@ app.use((err, req, res, next) => {
   });
 });
 
-
 // finally, let's start our server...
 // do not open a port on test environment,
 // this will be taken care of by mocha
 if (process.env.NODE_ENV !== 'test') {
-  const server = app.listen(process.env.PORT || 3000, () => {
+  server.listen(process.env.PORT || 3000, () => {
     /* eslint-disable no-console */
     console.log(`Listening on port ${server.address().port}`);
   });
 }
-module.exports = app;
+
+// store the socket server in a global variable to enable
+// use in different modules
+global.io = socketIO(server);
+
+socketIo(global.io);
+
+module.exports = {
```

Comment:
I commend your effort for coming up with the solution for this task. Can you make this export consistent with the rest of the codebase?

Reply:
Your feedback has been noted, but making use of es6 export syntax throws an error while exporting the server for testing.